.credo.exs

```diff
@@ -135,7 +135,7 @@
         {Credo.Check.Warning.ExpensiveEmptyEnumCheck, []},
         {Credo.Check.Warning.IExPry, []},
         {Credo.Check.Warning.IoInspect, []},
-        {Credo.Check.Warning.LazyLogging, []},
```

Comment:
Are the changes to the `.credo.exs` required for your changes to be valid?

Reply:
no, but those two are legacy checks that no longer apply and produce warnings in the logs if not disabled.

Reply:
Ah, good to know thanks!